source/_components/notify.simplepush.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Simplepush"
+description: "Instructions how to add Simplepush notifications to Home Assistant."
+date: 2016-09-11 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Notifications
+ha_release: 0.29
+---
+
+
+The `simplepush` platform uses [Simplepush](https://simplepush.io/) to delivery notifications from Home Assistant to your Android device.
+
+To add Simplepush to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - name: NOTIFIER_NAME
+    platform: simplepush
+    device_key: ABCDEFGHJKLMNOPQRSTUVXYZ
```

Comment:
Tested and keys I got are much shorter. Could be good to shorten it in example to avoid confusion.